ui/src/components/Menu.svelte

```diff
@@ -67,9 +73,9 @@
 
 </script>
 
-<div class="w-screen h-screen flex flex-row items-center justify-end" transition:fly={{ x: 400 }}>
+<div class="w-screen h-screen {isRightMargin ? 'mr-[5%]' : 'ml-[-70%]'} flex flex-row items-center justify-end" transition:fly={{ x: 400 }}>
   <!-- CONTROLS -->
-  <div class="w-[3.2vh] h-[85%] mr-[1vh] flex flex-col gap-[1vh]">
+  <div class="w-[3.2vh] h-[85%] {isRightMargin ? 'mr-[1vh]' : 'mr-[-27.4%]'} flex flex-col gap-[1vh]">
```

Comment:
For the controls to change side, you could go in the first div 

This: 
```
class="w-screen h-screen {isRightMargin ? 'mr-[5%]' : 'ml-[-70%]'} flex flex-row items-center justify-end"
```

Change the flex-row, to flex-row-reverse

To this: 
```
class="w-screen h-screen flex items-center { isRightMargin ? 'justify-end  flex-row' : 'justify-start  flex-row-reverse' }"
```